daemons/controld/controld_control.c

```diff
@@ -659,19 +659,30 @@ static pcmk__cluster_option_t controller_options[] = {
     {
         "no-quorum-policy", NULL, "select",
         "stop, freeze, ignore, demote, suicide", "stop", pcmk__valid_quorum,
-        "What to do when the cluster does not have quorum", NULL
+        N_("What to do when the cluster does not have quorum"), NULL
     },
     {
         XML_CONFIG_ATTR_SHUTDOWN_LOCK, NULL, "boolean", NULL,
         "false", pcmk__valid_boolean,
-        "Whether to lock resources to a cleanly shut down node",
-        "When true, resources active on a node when it is cleanly shut down "
+        N_("Whether to lock resources to a cleanly shut down node"),
+        N_("When true, resources active on a node when it is cleanly shut down "
             "are kept \"locked\" to that node (not allowed to run elsewhere) "
             "until they start again on that node after it rejoins (or for at "
-            "most shutdown-lock-limit, if set). Stonith resources and "
-            "Pacemaker Remote connections are never locked. Clone and bundle "
-            "instances and the promoted role of promotable clones are currently"
-            " never locked, though support could be added in a future release."
+            "most " XML_CONFIG_ATTR_SHUTDOWN_LOCK_LIMIT ", if set). Stonith "
```

Comment:
Bah, now I remember why this wasn't already like this. gettext can't handle constants in the string, it has to be a literal. We had to take constants out of a bunch of strings. There's no practical way around it, it's just a limitation of gettext.

Reply:
It runs before the C preprocessor??

Reply:
```
include/crm_internal.h:51:#  define _(String) gettext(String)
```

I don't see what would break here
```
$ grep -A1 'define gettext(' include/gettext.h
#  define gettext(Msgid) \
     dgettext (DEFAULT_TEXT_DOMAIN, Msgid)
--
# define gettext(Msgid) ((const char *) (Msgid))
# undef dgettext
```

Reply:
It's not the C side that breaks, but the utilities that create .po files for translation

Reply:
Oh okay, I wouldn't know how to test that. I'll take your word for it, no use going down that rabbit hole until I have to.

Reply:
It's possible to get around it in printf-style functions like "... %s ...", CONSTANT but that's not possible in initializations